### PR TITLE
Initial commit with basic Pythonnet 2.2 support

### PR DESCRIPTION
**Pull request by cowboygneox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+
+*Just add python sources and enjoy!*
+
+[https://hub.docker.com/r/pythondotnet/pythonnet/](https://hub.docker.com/r/pythondotnet/pythonnet/)
+
+## Current Support
+
+- Python 2.7 & 3.5
+- Mono 4.8.0
+- Pythonnet 2.2.0
```
Comment (vmuriart) [inline README.md]: `v2.2.2`

File: build_and_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+docker build -t pythondotnet/pythonnet:python2.7-mono4.8.0-pythonnet2.2.2 -f Dockerfile-python2.7-mono4.8.0-pythonnet2.2.2 .
+docker build -t pythondotnet/pythonnet:python3.5-mono4.8.0-pythonnet2.2.2 -f Dockerfile-python3.5-mono4.8.0-pythonnet2.2.2 .
+
+docker push pythondotnet/pythonnet:python2.7-mono4.8.0-pythonnet2.2.2
+docker push pythondotnet/pythonnet:python3.5-mono4.8.0-pythonnet2.2.2
```
Comment (vmuriart) [inline build_and_deploy.sh]: we can update this to `pythonnet` instead of `pythondotnet` now.

**Comment by cowboygneox** [comment]
I'll do updates in the AM.

**Comment by vmuriart** [comment]
no worries. thanks again for the help!

**Pull request merged**